src/crashlytics/getSampleCrash.ts

```diff
@@ -0,0 +1,59 @@
+import { Client } from "../apiv2";
+import { logger } from "../logger";
+import { FirebaseError } from "../error";
+import { crashlyticsApiOrigin } from "../api";
+
+const TIMEOUT = 10000;
+
+const apiClient = new Client({
+  urlPrefix: crashlyticsApiOrigin(),
+  apiVersion: "v1alpha",
+});
+
+export async function getSampleCrash(
+  projectId: string,
+  appId: string,
+  issueId: string,
+  variantId?: string,
+  sampleCount?: number,
+): Promise<string> {
+  try {
+    const queryParams = new URLSearchParams();
+    queryParams.set("filter.issue.id", `${issueId}`);
+    queryParams.set("page_size", String(sampleCount!));
```

Comment:
Don't use nonnull assertions on optional arguments. Either make this argument required, provide a default value, or wrap this in a conditional.